src/components/Profile.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ProfileRocketsDisplay from '../pages/ProfileRocketsDisplay';
+
+const Profile = () => (
+  <div>
+    <h1>My Rockets</h1>
+    <ProfileRocketsDisplay />
+  </div>
+);
```

Comment:
As required, The My Profile section should display all reserved rockets and space missions, Its currently blank after I reserved all missions and rocket, Kindly fix this 🙂 

![image](https://user-images.githubusercontent.com/66112760/151633786-ceb3c334-01e2-4e1c-a468-c8a2c8e0a1a5.png)

Reply:
changed